src/navigator.js

```diff
@@ -55,43 +55,52 @@ $.Navigator = function( options ){
-    //provide the id for the existing element
-    if( !options.id ){
-        options.id              = 'navigator-' + $.now();
-        this.element            = $.makeNeutralElement( "div" );
-        options.controlOptions  = {
-            anchor:           $.ControlAnchor.TOP_RIGHT,
-            attachToViewer:   true,
-            autoFade:         options.autoFade
-        };
-
-        if( options.position ){
-            if( 'BOTTOM_RIGHT' == options.position ){
-               options.controlOptions.anchor = $.ControlAnchor.BOTTOM_RIGHT;
-            } else if( 'BOTTOM_LEFT' == options.position ){
-               options.controlOptions.anchor = $.ControlAnchor.BOTTOM_LEFT;
-            } else if( 'TOP_RIGHT' == options.position ){
-               options.controlOptions.anchor = $.ControlAnchor.TOP_RIGHT;
-            } else if( 'TOP_LEFT' == options.position ){
-               options.controlOptions.anchor = $.ControlAnchor.TOP_LEFT;
-            } else if( 'ABSOLUTE' == options.position ){
-               options.controlOptions.anchor = $.ControlAnchor.ABSOLUTE;
-               options.controlOptions.top = options.top;
-               options.controlOptions.left = options.left;
-               options.controlOptions.height = options.height;
-               options.controlOptions.width = options.width;
+    //provide the id for the existing element or the element itself
+    if( !options.element ){
+        if( !options.id ){
+            options.id              = 'navigator-' + $.now();
+            this.element            = $.makeNeutralElement( "div" );
+            options.controlOptions  = {
+                anchor:           $.ControlAnchor.TOP_RIGHT,
+                attachToViewer:   true,
+                autoFade:         options.autoFade
+            };
+
+            if( options.position ){
+                if( 'BOTTOM_RIGHT' == options.position ){
+                options.controlOptions.anchor = $.ControlAnchor.BOTTOM_RIGHT;
+                } else if( 'BOTTOM_LEFT' == options.position ){
+                options.controlOptions.anchor = $.ControlAnchor.BOTTOM_LEFT;
+                } else if( 'TOP_RIGHT' == options.position ){
+                options.controlOptions.anchor = $.ControlAnchor.TOP_RIGHT;
+                } else if( 'TOP_LEFT' == options.position ){
+                options.controlOptions.anchor = $.ControlAnchor.TOP_LEFT;
+                } else if( 'ABSOLUTE' == options.position ){
+                options.controlOptions.anchor = $.ControlAnchor.ABSOLUTE;
+                options.controlOptions.top = options.top;
+                options.controlOptions.left = options.left;
+                options.controlOptions.height = options.height;
+                options.controlOptions.width = options.width;
+                }
             }
-        }
 
+        } else {
+            this.element            = document.getElementById( options.id );
+            options.controlOptions  = {
```

Comment:
It's not a big deal, I suppose, but it would be nice if this `options.controlOptions` code was shared between the `element` path and the `id` path, so we know the code will stay consistent in the future (without having to change things in two places). You could restructure to have a `if(options.id || options.element)` section.